ee/tabby-schema/src/schema/mod.rs

```diff
@@ -1030,7 +1032,6 @@ impl Mutation {
         ctx: &Context,
         input: SetPresetDocumentActiveInput,
     ) -> Result<bool> {
-        input.validate()?;
```

Comment:
why removing this?

Reply:
Besides - it seems we shall switch the input parameter of `SetPresetDocumentActiveInput` to ID instead of `name`

Reply:
Because some preset document can not pass this validate. Their name may contains "." and "-"

Reply:
Then we should modify the name regex instead?

> Besides - it seems we shall switch the input parameter of SetPresetDocumentActiveInput to ID instead of name
Besides - the name parameter shall be replaced with `id` instead.

Reply:
fixed.